effekt/shared/src/main/scala/effekt/source/Tree.scala

```diff
@@ -3,6 +3,12 @@ package source
 
 import effekt.context.Context
 import effekt.symbols.Symbol
+import kiama.util.{Position, Source, StringSource}
+
+import scala.{None as OptionNone, Some as OptionSome}
+
+import kiama.util.Position
+import scala.{Some => OptionSome , None => OptionNone }
```

Comment:
This seems very duplicated?
Also, why are the `OptionSome` and `OptionNone` needed?

Reply:
There is a clash with `Maybe.Some`, `Maybe.None` otherwise. Maybe there's a better way to resolve that?